benchmark/checks/check_multiagent.py

```diff
@@ -158,8 +158,8 @@ def run(
     runner = AEARunner(agents, runner_mode)
     runner.start(threaded=True)
 
-    for agent in agents:
-        wait_for_condition(lambda: agent.is_running, timeout=5)
+    for agent_ in agents:
+        wait_for_condition(lambda a=agent_: a.is_running, timeout=5)
```

Comment:
this was a bug!

Reply:
actually not a bug, it's used inplace. so closure is not needed.
but automatic check can suppose closure is needed.